ansible-playbook-base/roles/passwdless-ssh-setup/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# This task determines the number of bricks needed and configures
+# the bricks on the respective server hosts
+
+- name: Generate ssh keys
+  user:
+    name: root
+    generate_ssh_key: yes
+    ssh_key_file: .ssh/id_rsa
+
+- name: Copy ssh public key
+  shell: cat ~/.ssh/id_rsa.pub
+  register: ssh_keys
+
+# TODO: Deply server host keys in servers only and same with client
```

Comment:
Open an issue if this is not going to be addressed with this PR

Reply:
Another thought is to separate this into groups clients and servers, that way we do not need to track a TODO for later